config.py

```diff
@@ -100,6 +102,37 @@ def validate_config(config):
             ):
                 raise ConfigError("Config error: Ignored tables parameter should be a list")
 
+    if "notification" in config:
+        settings = [
+            "smtp_server",
+            "smtp_username",
+            "smtp_password",
+            "email_sender",
+            "email_subject",
+            "email_recipients",
+        ]
+
+        for setting in settings:
+            if setting not in config.notification:
+                raise ConfigError(f"Config error: `{setting}` is missing from `notification`.")
+
+        if not isinstance(config.notification.email_recipients, list):
+            raise ConfigError("Config error: `email_recipients` should be list of address.")
+
+        smtp_conn: smtplib.SMTP = None
+
+        try:
+            smtp_conn = smtplib.SMTP(config.notification.smtp_server)
+        except socket.gaierror:
+            raise ConfigError(f"Config error: Cannot connect to SMTP server: `{config.notification.smtp_server}`.")
+
+        try:
+            smtp_conn.login(config.notification.smtp_username, config.notification.smtp_password)
+        except smtplib.SMTPAuthenticationError:
+            raise ConfigError(
+                f"Config error: Cannot login to SMTP server with user: `{config.notification.smtp_username}` and a specified password."
+            )
```

Comment:
I think connecting to SMTP server does not fully tell whether sending email will be successful (e.g. server could only admit some "From:" headers). To help admins check that email notifications are correctly set up, I would suggest to have a special command to send a test mail, e.g. `python dbsync_daemon.py --test-notification-email` - and it would send an email and exit, so that admin does not need to try to break sync to verify that notifications work.